lib/perimeterx.js

```diff
@@ -68,32 +67,37 @@ function pxMiddleware(req, res, next) {
             });
         } else {
             pxutil.verifyUserScore(pxCtx, (action) => {
-                pxLogger.debug('score action ' + action);
-                if (action === pxconfig.SCORE_EVALUATE_ACTION.COOKIE_PASS_TRAFFIC) {
-                    pxLogger.debug('sending page requested activity from cookie');
-                    pxCtx.passReason = pxconfig.PASS_REASON.COOKIE
-                    return pxPass(pxCtx, next);
+                let isTerminated = false;
```

Comment:
for my understanding, `ADDITIONAL_ACTIVITY_HANDLER`  function is just to send activities using an additional handler, why would it return boolean value of terminated?